fastlane/Constants/Constant.swift

```diff
@@ -17,12 +17,66 @@ enum Constant {
 
     // MARK: - Project
 
-    static let bundleId = "<#bundleId#>"
+    static let bundleId: String = "<#bundleId#>"
     static let productName: String = "<#productName#>"
     static let scheme: String = "<#scheme#>"
 
     // MARK: - Path
 
-    static let derivedDataPath = "./DerivedData"
-    static let buildPath = "./Build"
+    static let derivedDataPath: String = "./DerivedData"
+    static let buildPath: String = "./Build"
+
+    // MARK: - Firebase
+
+    static let firebaseAppIdStaging: String = "<#firebaseAppIdStaging#>"
+    static let firebaseAppIdProduction: String = "<#firebaseAppIdProduction#>"
+    static let firebaseTesterGroups: String = "<#group1#>, <#group2#>"
+}
+
+extension Constant {
+
+    enum Environment: String {
+
+        case staging = "Staging"
+        case production = ""
+
+        var productName: String { "\(Constant.productName) \(rawValue)".trimmed }
+
+        var scheme: String { "\(Constant.scheme) \(rawValue)".trimmed }
+
+        var bundleId: String {
+            let bundleId = Constant.bundleId
+            switch self {
+            case .staging: return "\(Constant.bundleId).\(rawValue.lowercased())"
```

Comment:
I think we might need to create a bundle id constant for staging as well, not every time we have
- product: co.nimblehq.app
- staging: co.nimblehq.app.staging 

We might have something like `co.nimblehq.app-staging` for example.

What do you think?

Reply:
@suho Yes, I also think it's better with explicit define. Updated in [1c5b9d7](https://github.com/nimblehq/ios-templates/pull/354/commits/1c5b9d7fcc401f4a9a004872b69ead50a7bd6e4c)